payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/WalletServiceImpl.java

```diff
@@ -37,4 +43,41 @@ public void deleteWallet(final long id) {
         walletRepository.deleteById(id);
     }
 
+    @Transactional
```

Comment:
what type of isolation level do you use and why?

Reply:
By default postgres use READ_COMMITED(I should write it explicitly), thanks to that before doing any UPDATE operation we take the most recent data